src/map/battle.cpp

```diff
@@ -6138,6 +6138,13 @@ static int32 battle_calc_attack_skill_ratio(struct Damage* wd, struct block_list
 			RE_LVL_DMOD(100);
 			skillratio += skillratio * (20 * (sd ? pc_checkskill(sd, WH_ADVANCED_TRAP) : 5)) / 100;
 			break;
+		case WH_WILD_WALK:
+			skillratio += -100 + 1800 + 2800 * skill_lv;
+			skillratio += 5 * sstatus->con;
```

Comment:
Ignore this comment if you know the con and other skills ratio, otherwise please add a comment, like
```suggestion
			// !TODO: unknown con and WH_NATUREFRIENDLY/HT_STEELCROW skills ratio
			skillratio += 5 * sstatus->con;
```